### PR TITLE
Prevent notices on JSON_LD

### PR DESCRIPTION
**Pull request by bordoni**

### CODE REVIEW

**Comment by borkweb** [comment]
🌍

File: src/Tribe/JSON_LD/Organizer.php [hunk 1]
```diff
@@ -41,6 +41,11 @@ public static function instance( $name = null ) {
 	public function get_data( $post = null, $args = array( 'context' => false ) ) {
 		$data = parent::get_data( $post, $args );
 
+		// If we have an Empty data we just skip
+		if ( empty( $data ) ) {
+			return $data;
```
Comment (zbtirrell) [inline src/Tribe/JSON_LD/Organizer.php]: I'd prefer to return 'array()' here

File: src/Tribe/JSON_LD/Venue.php [hunk 2]
```diff
@@ -42,6 +42,11 @@ public static function instance( $name = null ) {
 	public function get_data( $post = null, $args = array( 'context' => false ) ) {
 		$data = parent::get_data( $post, $args );
 
+		// If we have an Empty data we just skip
+		if ( empty( $data ) ) {
+			return $data;
```
Comment (zbtirrell) [inline src/Tribe/JSON_LD/Venue.php]: And here

**Comment by bordoni** [comment]
@zbtirrell fixed what you suggested.

**Pull request merged**